nss-day-cohort-39/github-workflow-aquarium-cohort-39-lionfish
# New additon

Changes
- Added a lionfish to index.html
- Added dialog button to dialog.js

How to test
- Make sure lionfish shows up with the previous original fish
- click dialog button and see if it fully functions with fish's info
